### PR TITLE
Task/ar/improve docs ad 1110

### PR DESCRIPTION
**Pull request by monstrorivas**
This PR adds more verbiage to the installation instructions for help users figure out where to get the values required for the config file. 

Can you please proof-read section [User Installation (Seeq Data Lab)](https://github.com/seeq12/seeq-azureml/tree/task/ar/improve-docs-AD-1110#user-installation-seeq-data-lab) and provide comments/edits/etc.?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,12 +110,42 @@ The latest build of the project can be found [here](https://pypi.org/project/see
 as a wheel file. The file is published as a courtesy and does not imply any guarantee or obligation for support from the
 publisher.
 
+### Pre-requisites
+
+Before installing the **seeq-azureml** Seeq Add-on, you will have to create an **Azure application** and **service
+principal**. Follow the steps
+in [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to
+create your application, and
+choose [Option 2](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret)
```
Comment (anvenkat18) [inline README.md:119]: Is it worth mentioning if option 1 is viable or won't work at all?

File: README.md [hunk 2]
```diff
@@ -110,12 +110,42 @@ The latest build of the project can be found [here](https://pypi.org/project/see
...
+
 1. Create a **new** Seeq Data Lab project and open the **Terminal** window
 2. Run `pip install seeq-azureml`
 3. Run `python -m seeq.addons.azureml [--users <users_list> --groups <groups_list>]`
 4. Create an `aml_config.ini` file in the `~/.seeq` folder of the Seeq Data Lab Project with the information required to
-   connect to the Azure ML services. The file must contain the options specified
-   in [here](https://github.com/seeq12/seeq-azureml/tree/master/aml_config.ini)
+   connect to the Azure ML services. The following steps summarized how to create the `aml_config.ini` file in the
+   correct Seeq Data Lab location and where to get the values required for the configuration.
+   1. Download the `aml_config.ini` from [here](https://github.com/seeq12/seeq-azureml/tree/master/aml_config.ini)
```
Comment (anvenkat18) [inline README.md]: Suggest: Download the template `aml_config.ini` from

**Review by anvenkat18 — APPROVED** [review]
Looks good. Couple of minor comments. I'll defer to @cherrera2001 for the integration details.

File: README.md (hunk 1 above)
Comment (cherrera2001) [reply README.md:119]: I think the description currently is clear enough. What I have learned from doing this in the past is that detailing what not to do, can a lot of times lead to more confusion.

**Review by cherrera2001 — APPROVED** [review]
LGTM!

**Pull request merged**